ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-layering.yaml

```diff
@@ -7,6 +7,10 @@ base_images:
     name: builder
     namespace: ocp
     tag: rhel-8-golang-1.17-openshift-4.10
+  rhel-coreos-base:
+    name: rhel-coreos-base
```

Comment:
This should probably be `name: rhel-coreos` since the image is called `rhel-coreos` in the `rhcos-devel` namespace. The key above (`rhel-coreos-base:`) provides an identifier for this image within the CI Imagestream.

Reply:
Fixed!